packages/migration/migrations/hearth/20221031131452-add_practioner_id_event_registrations.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { query, writePoints } from './../../utils/influx-helper.js'
+
+export const up = async (db, client) => {
+  const session = client.startSession()
+  await session.withTransaction(async () => {
+    const resultBirth = await query(
+      "SELECT * FROM birth_registration WHERE registrarPractitionerId = ''"
+    )
+    const birthPromises = updateInflux(resultBirth, 'birth_registration', db)
+    await Promise.all(birthPromises)
+
+    const result = await query(
+      "SELECT * FROM death_registration WHERE registrarPractitionerId = ''"
+    )
+    const deathPromises = updateInflux(result, 'death_registration', db)
+    await Promise.all(deathPromises)
+  })
+}
+
+const updateInflux = (result, measurement, db) => {
+  return result.map(
+    async ({
+      compositionId,
+      time,
+      ageInDays,
+      ageInYears,
+      currentStatus,
+      deathDays,
+      ...tags
+    }) => {
+      const task = await db.collection('Task').findOne({
+        focus: {
+          reference: `Composition/${compositionId}`
+        },
+        businessStatus: {
+          coding: [
+            {
+              system: 'http://opencrvs.org/specs/reg-status',
+              code: 'REGISTERED'
+            }
+          ]
+        }
+      })
```

Comment:
This way we won't be able to find the tasks for the declarations that have been CERTIFIED for example. (This applies for ARCHIVED as well)